os/pm/Kconfig

```diff
@@ -45,6 +45,15 @@ config PM_METRICS_DURATION
 
 endif
 
+config PM_MAX_WAKEUP_TIMER
```

Comment:
how about PM_MAX_STATIC_TIMER ?

And this pr is having both pm_wakeup_timer and pm_timer.
How about unifying the terms into one?